content/introduction/getting-started-with-sdks/index.mdx

```diff
@@ -0,0 +1,485 @@
+[API Playground](https://sharetribe.GitHub.io/flex-sdk-js/try-it-in-the-playground.html).
+For a more detailed guide on using the SDK with code, keep following
+this tutorial.
+
+## Installation
+
+To get started with the SDKs, you'll first need to install them with
+either [Yarn](https://yarnpkg.com/)
+
+<Tabs items={['yarn', 'npm']}>
+  <Tabs.Tab>
+  `yarn add sharetribe-flex-sdk`
+
+`yarn add sharetribe-flex-integration sdk`
+
+  </Tabs.Tab>
+  <Tabs.Tab>
+  `npm install sharetribe-flex-sdk`
+
+`npm install sharetribe-flex-integration sdk`
+
+  </Tabs.Tab>
+</Tabs>
+
+## SDK Initialization Overview
+
+### Obtaining the Client ID and secret
+
+To use the Marketplace API or Integration API, you will need a **client
+ID** from an application. You can create an application in Sharetribe
+Console: _Build > Advanced > Applications_
+
+![Sharetribe Console: Applications tab](./console-build-application.png)
+
+You should also make note of the **client secret**, as some requests
+require a secure context and this can only be done when initializing the
+SDK with both the client ID and secret.
+
+An example of this type of request is making a privileged transition in
+a transaction. This requires that the:
+
+- Client ID and secret are used when creating an instance of the SDK
+- A token exchange happens to obtain a trused user access token
+- The request is made from a secure context.
+
+<Callout type="info">
+  In the [Sharetribe Web
+  Template](https://github.com/sharetribe/web-template), these types of
+  requests are made on an small Node server included in the Template.
+</Callout>
+
+### Initializing a basic SDK instance
+
+You can initialize a basic SDK instance using only the application's
+Client ID. Ensure the client ID is from a **Marketplace API
+application** created in Console.
+
+```jsx
+const sharetribeSdk = require('sharetribe-flex-sdk');
+
+const sdk = sharetribeSdk.createInstance({
+  clientId: '<Client ID>',
+});
+```
+
+This will allow you to make requests that don't require authentication
+or a trusted SDK instance, e.g. querying listings.
+
+### Initializing an Integration SDK instance
+
+If you are accessing Integration API, you'll need to also provide the
+client secret. Ensure the client ID is from an **Integration API
+application** created in Console.
+
+```jsx
+const sharetribeIntegrationSdk = require('sharetribe-flex-integration-sdk');
+
+const integrationSdk = sharetribeIntegrationSdk.createInstance({
+  clientId: '<Client ID>',
+  clientSecret: '<Client secret>',
+});
+```
+
+### Initializing a trusted SDK instance
+
+To make certain requests, you will need to access the APIs from a
+trusted context. In order to do this, you'll need to log in using the
+SDK and perform a token exchange to retrieve a trusted token.
+
+<Steps>
+
+## Create an instance of the SDK with the client ID, secret, and token store
+
+```jsx
+// use 'sharetribe-flex-integration-sdk' here to create and integration SDK instance
```

Comment:
This comment can be removed, because the trusted exchange only works for Marketplace API (Integration API is a trusted context by default and doesn't have user logins)